Source/WebCore/platform/graphics/skia/ComplexTextControllerSkia.cpp

```diff
@@ -151,28 +151,41 @@ static std::optional<HBRun> findNextRun(std::span<const UChar> characters, unsig
     return std::optional<HBRun>({ startIndex, textIterator.currentIndex(), currentScript.value() });
 }
 
+static inline void forEachHBRun(const std::span<const UChar>& characters, bool ltr, std::function<void(const HBRun&)> callback)
```

Comment:
```suggestion
static inline void forEachHBRun(const std::span<const UChar>& characters, bool ltr, Function<void(const HBRun&)>&& callback)
```

Reply:
I think we save a copy using Function from WTF that is non-copyable and passing an rvalue ref.